docs/user-guide.md

```diff
@@ -98,10 +98,12 @@ for i in *.md; do grep -r $i . | grep -v "^\./$i" > /dev/null; rv=$?; if [[ $rv
   Provides resource information such as size, type, and quantity to assist in
   assigning Kubernetes resources appropriately.
 
+* **CLI roadmap** ([cli-roadmap.md](cli-roadmap.md)): Kubernetes
```

Comment:
I think it rather needs to be in the design or devel subdirectory, but I thought someone placed it here on purpose.

Where should I create a link to this document? I think all documents should be linked somewhere, otherwise people may miss them.